src/pages/home/report/ReportActionItemSingle.js

```diff
@@ -51,13 +52,14 @@ const showUserDetails = (email) => {
 };
 
 const ReportActionItemSingle = (props) => {
+    const actorEmail = props.action.actorEmail.replace(CONST.REGEX.MERGED_ACCOUNT_PREFIX, '');
```

Comment:
Not directly related, but we should have created a utility function that did this replacement. Later on, at some places we forgot to do this replacement which caused https://github.com/Expensify/App/issues/32127.